Sources/AWSLambdaEvents/LambdaProxyEvent.swift

```diff
@@ -0,0 +1,88 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2017-2022 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+public struct LambdaProxyEvent: Codable {
+    public struct Context: Codable {
+        public struct HTTP: Codable {
+            public let httpMethod: HTTPMethod
+            public let path: String
+            public let `protocol`: String
+            public let sourceIp: String
+            public let userAgent: String
+        }
+
+        /// Authorizer contains authorizer information for the request context.
+        public struct Authorizer: Codable {
+            public let claims: [String: String]?
+            public let scopes: [String]?
+        }
+
+        public let accountId: String
+        public let apiId: String
+        public let domainName: String
+        public let domainPrefix: String
+        public let stage: String
+        public let requestId: String
+        
+        public let httpMethod: HTTPMethod
+        public let authorizer: Authorizer?
+
+        public let resourcePath: String?
+        public let path: String?
+        
+        /// The request time in format: 23/Apr/2020:11:08:18 +0000
+        public let time: String?
+        public let requestTime: String?
+        public let requestTimeEpoch: UInt64
```

Comment:
Using correct casing for ID (more context: https://developer.apple.com/documentation/swift/identifiable, https://www.swift.org/documentation/api-design-guidelines/#follow-case-conventions) will require us to use explicit CodingKeys.